g3/generate/commit/prompts/creator.py

```diff
@@ -47,17 +48,22 @@ def user_messages(self) -> list:
     def create_system_messages(
         self, tone: MessageTone, jira: Optional[str] = None, include: Optional[str] = None
     ) -> list:
-        content = f"""You are a helpful assistant creating commit titles and commit descriptions for git commits in a
- software engineering team based on their stashed git changes which will be provided to you. We defined a good commit
- title as one that explains what was changed. The commit description if asked should contain what was changed and why
- the change was made. Don't include the provided code in your response. Also, dont include in the response the words
- commit title, or commit description. The response will be directly used by git commit command. The goal for the commit
- title is to briefly complete a sentence which starts like 'This commit if applied will:' but dont include this sentence
- in your response. Be brief on your response about the commit title using less than 80 characters, use imperative and
- start with a verb. Whatever follows should be the commit title that you suggest. The commit description if asked,
- should be a few bullets which will contain short descriptions about the changes using at most
- {config.commit_description_max_words} words. If the words provided are 0 then you should not include a commit
- description. Use the following tone when creating the commit message: {tone.value}."""
+        content = f"""You are a helpful assistant creating commit titles and commit descriptions for git commits in a software engineering team based on their stashed git changes which will be provided to you. 
+ 
+ We defined a good commit title as one that explains what was changed. 
```

Comment:
Are the empty lines intentional? Please also execute `pre-commit install`, the lines should be up to 120 chars

Reply:
fixed!